### DISCUSSION TITLE
How to unified PTH and NPTH files with Gerber drill format job?

### DISCUSSION THREAD

**Discussion by BetCvR6** [INTI-CMNB/KiBot#654] [Q&A] [answered]
Hi,
Is there a mean to unified PTH and NPTH files in one file with Gerber drill format job?
In job's documentation, we can read :
### pth_id : [string] Force this replacement for %i when generating PTH and unified files.
But I can't do it after several tests...
Does anybody already did this?
Thanks.

**Comment by set-soft** [comment, marked as answer]
Hi @BetCvR6 !

I'm afraid this isn't possible for the gerber format. Use Excellon format for unified drill files.

**Reply by BetCvR6** [reply]
Hi @set-soft!

It's okay for me, this works and makes the job.
Thanks a lot!
